tests/unit/states/test_jboss7.py

```diff
@@ -12,6 +12,7 @@
 # Import Salt libs
 import salt.states.jboss7 as jboss7
 from salt.exceptions import CommandExecutionError
+from six import text_type
```

Comment:
Please change this to `from salt.ext import six` and use `six.text_type()` instead of `text_type()`. It wouldn't cause any issues here, but I'd like to keep the code uniform through the codebase, so we don't run the risk of another contributor seeing your example here and then writing a state/execution/etc. module using the same technique. Our loader will gobble up all public functions in the module's global namespace, so doing this in an execution module would add `modname.text_type` to the `__salt__` dunder, and we don't want that.